File: Domain/Models/Vehiculos/TipoVehiculo.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace Domain.Models.Vehiculos
{
    public class TipoVehiculo
    {
        [Key]
        public int TipoVehiculoId { get; set; }

        [Column(TypeName = "nvarchar(20)")]
        public string Descripcion { get; set; }

    }
}

File: CarAgency/ClientApp/src/domain/models/vehiculos/TipoVehiculo.ts
import ITipoVehiculo from "domain/interfaces/vehiculos/ITipoVehiculo";

export class TipoVehiculo implements ITipoVehiculo{
    constructor(){
        this.TipoVehiculoId = 0;
        this.Descripcion = "";        
    }

    TipoVehiculoId: number;
    Descripcion: string;

}